source/draft_users.rst

```diff
@@ -278,9 +278,16 @@ Delete a user. Returns a 200 OK with empty response body on success, or an
 Role Management
 ---------------
 
-Role management is handled through the ``projects`` endpoints. The projects
-model contains a ``users`` object, which contains three lists: ``members``,
-``spectators``, and ``managers``, each lists of usernames. An admin or project
-manager may set these at any time, adding to or removing from any of the lists.
-A project must always have at least one manager, however. Attempting to remove
-all managers from a project will return an error.
+Role management is handled through the ``projects`` and ``users`` endpoints.
+
+The users model contains the ``site_spectator``, ``site_manager``, and ``site_admin``
```

Comment:
Fair enough, we do use this language, though it is mostly confined to the draft_models doc. This just reads a little more like a timesync-node description to me than a pure abstract API description